pyipmi/messaging.py

```diff
@@ -96,6 +96,30 @@ def set_user_access(self, userid, ipmi_msg, link_auth, callback_only,
         rsp = self.send_message(req)
         check_completion_code(rsp.completion_code)
 
+    def set_user_password(self, userid, password=''):
+        if len(password) > 20:
```

Comment:
why 20 if you use ljust(16)?

Reply:
Oh you're right, my bad. I mixed up between IPMI v1.5 and v2.0